docs/pages/getting-started/docker-compose.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting started with Teleport using Docker Compose
+description: How to get started with Teleport Open Source Edition using Docker Compose locally.
+h1: Get started with Docker Compose
+---
+
+This section will cover:
+
+- Getting started with a local Teleport cluster using docker compose.
+- Using Teleport with OpenSSH, ansible and Teleport's native client, `tsh`.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
+- Docker v(=docker.version=) or later and [`docker-compose`](https://docs.docker.com/compose/install/) v(=docker.compose.version=) or later.
+
+```bash
+docker-compose version
+# docker-compose version (=docker.compose.version=), build unknown
+
+docker version
+#Client: Docker Engine - Community
+# Version:           (=docker.version=)
+```
+
+## Step 1/3. Start teleport demo lab
+
+Let's use `docker-compose` to start Teleport demo lab - a configured local cluster:
+
+```bash
+# Download the quick start file from our Github repo
+curl -Lso teleport-lab.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-lab.yml
+# Start teleport demo lab using docker-compose
+docker-compose -f teleport-lab.yml up
+```
+
+<Admonition type="note" title="Note">
+You can later stop the Teleport lab using:
+
+```bash
+docker-compose -f teleport-quickstart.yml down
+```
+</Admonition>
+
+## Step 2/3. Explore teleport using CLI
+
+Let's jump into container with setup clients and explore Teleport:
+
+```bash
+docker exec -ti term /bin/bash
+```
+
+<Admonition type="note" title="Note">
+We will run all future commands from the `term` container.
+</Admonition>
+
+Welcome to Teleport Lab. With Teleport you can access servers, databases and web apps in your cluster.
+
+Let's Try a couple of commands to get started.
+Teleport speaks SSH. You can SSH into it using OpenSSH:
+
+```bash
+ssh root@luna.teleport
```

Comment:
I get a TOFU prompt when I run this:

```bash
~/dev/teleport/docker(sasha/demo*) » docker exec -ti term /bin/bash
Initializing new SSH agent...
succeeded
Identity added: bot (bot)
Certificate added: bot-cert.pub (bot)
Welcome to Teleport Lab.

Access servers, databases and web apps in your cluster securely with Teleport.
Try a couple of commands to get started.

Teleport speaks SSH. You can SSH into it using OpenSSH:

ssh root@luna.teleport

Teleport is a bastion server for your OpenSSH hosts. SSH into OpenSSH server and record all commands:

ssh root@mars.openssh.teleport

Run ansible on Teleport nodes and OpenSSH servers:

cd /etc/teleport.d/ansible && ansible all -m ping

Try Teleport's client command: tsh. It's like SSH, but with superpowers.
Find all hosts matching label env=example and run hostname command:

tsh ssh root@env=example hostname
container(term) /mnt/shared/certs: ssh root@luna.teleport
The authenticity of host 'luna.teleport (<no hostip for proxy command>)' can't be established.
RSA key fingerprint is SHA256:Q0E9S3ZNhNwR+tQIkTFp8dzvSpbMsjRA0oXGy/DvzCk.
Are you sure you want to continue connecting (yes/no/[fingerprint])?
```

Looks like the hostnames aren't configured correctly in `~/.ssh/known_hosts` - we should fix this to avoid all the TOFU prompts.

This edit worked for me:
```
@cert-authority luna.teleport,*.luna.teleport,*.openssh.teleport ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKRo0SRrafW+3cui3yBzsFpvt6hXx74aEc6Y1XOmC4stpmHQ54NY+5S/TCVZNhIhHq2VbyfUcGTSn8zMjoWIg/PvP989foE2axW+PEVktHwTBrJb1muVNBBJzQ+8l6Ke98sONRQnNfMv0/J2RlDLK0BgMI48JvFTXpPEl56vflROTCVmbyPxCHhT17TJTG9jk9B67lSWN/lLyJXMOfzTuGi2n0glkKfEtL8Zmwi7hBnKAAsonXgDpS47tZMpoGGrL9PBcWdpL4IIUBHcyj8HjwTzsrWOwP3boj/3AkZWxqkpc36rPUUDw4Uc6OSEh+Z7QnSQ2l6CJMhnwCjBcYtS3d type=host
```

Reply:
Fixed, thanks

Reply:
I'm still getting a TOFU prompt:

```
container(term) /mnt/shared/certs: cat ~/.ssh/known_hosts
@cert-authority *.luna.teleport ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDKRo0SRrafW+3cui3yBzsFpvt6hXx74aEc6Y1XOmC4stpmHQ54NY+5S/TCVZNhIhHq2VbyfUcGTSn8zMjoWIg/PvP989foE2axW+PEVktHwTBrJb1muVNBBJzQ+8l6Ke98sONRQnNfMv0/J2RlDLK0BgMI48JvFTXpPEl56vflROTCVmbyPxCHhT17TJTG9jk9B67lSWN/lLyJXMOfzTuGi2n0glkKfEtL8Zmwi7hBnKAAsonXgDpS47tZMpoGGrL9PBcWdpL4IIUBHcyj8HjwTzsrWOwP3boj/3AkZWxqkpc36rPUUDw4Uc6OSEh+Z7QnSQ2l6CJMhnwCjBcYtS3d type=host
container(term) /mnt/shared/certs: ssh root@luna.teleport
The authenticity of host 'luna.teleport (<no hostip for proxy command>)' can't be established.
RSA key fingerprint is SHA256:Q0E9S3ZNhNwR+tQIkTFp8dzvSpbMsjRA0oXGy/DvzCk.
Are you sure you want to continue connecting (yes/no/[fingerprint])? ^C
```